tensorflow/compiler/xla/service/gpu/cudnn_conv_rewriter.cc

```diff
@@ -258,15 +278,6 @@ MatchBackwardInput(HloInstruction* conv) {
   const auto no_match_result =
       std::make_tuple(false, Window(), ConvolutionDimensionNumbers(), nullptr);
 
-  // TODO(b/119479517): Theoretically cuDNN supports grouped convolutions also
```

Comment:
Can you please leave this out of this change and just work on the BackwardFilter depthwise convolution?
It would be better to have this in a separate CL so the impact can be separately checked when running our benchmarks.

Reply:
I have reverted this change and adding another separate PR